BPRBE/UserInterface/Enums/ViolationSeverity.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace BPRBE.Models.UI.Results.Enums;
+﻿namespace BPRBE.UserInterface.Enums;
```

Comment:
Don't we want the backend to be blind about the FE? I would suggest we keep "UI" or anything like that outside of BE. If anything we'll use the enums in our processing as well. It's safe to just call the namespace BPRBE.Enums if you ask me

Reply:
Yeah I was a bit on the fence with this too. I figured to keep them in this project rather than the blazor project as these enums and models were not related to specifically Blazor but could be used for any UI.
But yeah they can probably be used in the BE as well